var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,26 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('serial', default=False, description='Disable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', when='+mpi', type=('link'))
+
+    # this will disable mpi
+    def configure_args(self):
+        spec = self.spec
+        args = []
+        if '+serial' in spec:
+            args += ['--enable-serial']
+        elif '~mpi' in spec:
+            args += ['--enable-serial']
+        elif '-mpi' in spec:
+            args += ['--enable-serial']
```

Comment:
Also, what happens when I build `meme +mpi +serial` or `meme -mpi -serial`? I don't know anything about `meme`, but I think you need a single variant to control whether or not you build MPI support.